test/unit/range/view/view_pairwise_combine_test.cpp

```diff
@@ -0,0 +1,506 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
```

Comment:
Can you add a test that verifies all the Range concepts?

Reply:
Which range concepts do you mean?